.github/workflows/issue-types.yml

```
@@ -0,0 +1,51 @@
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  sync-issue-types:
+    runs-on: ubuntu-latest
+    name: Sync Issue Types
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)